src/plugins/kernel_install/debian.sh

```diff
@@ -36,9 +55,42 @@ function update_boot_loader()
 
   # Update grub
   cmd="$sudo_cmd grub-mkconfig -o /boot/grub/grub.cfg"
+  if [[ "$local" == 'vm' ]] ; then
+    vm_update_boot_loader "$name" "$cmd"
+    exit
```

Comment:
I "exited" because the process is complete and the remain operations after that are related to local deploy and not vm deploy (ie: I want to skip everything after that point).